src/upload.php

```diff
@@ -29,11 +29,13 @@
 
 // check file size ** final size TBD **
 
-if ($_FILES['file']['size'] > 500000) {
+if ($_FILES['file']['size'] > 5000000) {
```

Comment:
Rather than `5000000`, we should be using the size allowed by the server. It's 200MB. Is this value bytes or kilobytes?

Reply:
its bytes.